tests/ui/manual_retain.rs

```diff
@@ -215,6 +224,12 @@ fn vec_deque_retain() {
     bar = foobar.into_iter().filter(|x| x % 2 == 0).collect();
 }
 
+#[clippy::msrv = "1.70"]
+fn _msrv_170() {
+    let mut binary_heap = BinaryHeap::from([1, 2, 3]);
+    binary_heap = binary_heap.into_iter().filter(|x| x % 2 == 0).collect();
+}
```

Comment:
This should be 1.69 to test that it isn't applied when the MSRV is below `BinaryHeap::retain`'s introduction

Reply:
Thanks, I updated it at [1eff39d](https://github.com/rust-lang/rust-clippy/pull/11329/commits/1eff39ddb61ebeba58446894b8bb238840d5cc06)